src/components/localization/localized-link.js

```diff
@@ -186,12 +185,10 @@ const ExternalLink = ({
     type,
     ...props
 }) => {
-    const { is_eu_country } = useContext(DerivStore)
```

Comment:
We still need the modal pop for EU when it is an external link (not deriv product links). You need to move smart trader link onto the array that doesn't have the EU modal.

Reply:
Should this not be removed anymore then? @pavel-latyshou-deriv